habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -842,7 +850,8 @@ def step(self, action: Union[str, int]) -> Observations:
             )
             self.add_perf_timing("get_sensor_observations", t_start)
 
-        if self._enable_gfx_replay_save:
+        # TODO: Support recording while batch rendering
+        if self._enable_gfx_replay_save and not self._batch_render:
```

Comment:
We definitely need to clean this up in July. Super-confusing that when both of these are set to true, we don't save any keyframes. You should document this in your markdown doc.